fga-eps-mds/2020.1-stay-safe-front-end
# Dark mode improvements

## Issue 
Resolves #91 


## Description

This PR fixes visual impairments on the search in the cities drop down, also made the main theme dark mode and made the user's choice to be stored
## Screenshots (if appropriate):



## Checklist:



## Known problems (if appropriate):



## How to validate?

Check the color of the text on the search in the cities dropdown on dark mode, uninstall expo from cellphone and install it again to check if the main theme is now dark mode, then proceed to swap to light mode, close and open the app to verify if its still on the light mode
